sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/DoFnTester.java

```diff
@@ -372,11 +383,15 @@ public T apply(WindowedValue<T> input) {
    * Returns the value of the provided {@link Aggregator}.
    */
   public <AggregateT> AggregateT getAggregatorValue(Aggregator<?, AggregateT> agg) {
+    return extractAggregatorValue(agg, agg.getCombineFn());
```

Comment:
Seems appropriate to pull out the `CombineFn` inside `extractAggregatorValue`, yea? Is there any meaning to passing any other value for this second parameter?